contents/blog/introducing-hogql.md

```diff
@@ -0,0 +1,112 @@
+
+To give you a sense of what's possible with SQL, here's an example query we use to summarize data from our from [our 404 page](/hedgehogsgalore) and display which countries prefer their pizzas with pineapple on them.
+
+```
+   select properties.$geoip_country_name,
+          if(countIf(not properties.does_pineapple_go_on_pizza) > countIf(properties.does_pineapple_go_on_pizza),
+            '🍅 Does not belong on pizza',
+            if(countIf(not properties.does_pineapple_go_on_pizza) = countIf(properties.does_pineapple_go_on_pizza),
+              '🥦 It is a tie',
+              '🍍 Belongs on pizza'
+          )) as Result,
+          concat(
+             repeat('🍍', countIf(properties.does_pineapple_go_on_pizza)),
+             repeat('🍅', countIf(not properties.does_pineapple_go_on_pizza))
+          ) as Answers,
+          count() as `Number of answers`,
+          concat(
+            toString(round(countIf(properties.does_pineapple_go_on_pizza) / count() * 1000) / 10), 
+            '%'
+          ) as `Percentage pineapple`
+     from events
+    where event = 'pineapple_on_pizza_survey'
+ group by properties.$geoip_country_name
+ order by count() desc
+    limit 100
+```
+
+## The PostHog event explorer 
+We haven't introduced HogQL in isolation - we've also bought it into existing parts of PostHog, supercharging them to deliver new functionality. 
+
+The Event Explorer (previously the Live Events tab) is a great example of this. What was previously a stream of incoming events now offers wholly new ways to interact with your data and filter it into custom tables. 
+
+The event explorer enables you to view and edit the JSON source behind the table, so you can create tables in PostHog and add them to dashboards. You can also use HogQL expressions (e.g. `properties.$screen_width * $properties.screen_height`) and aggregations (e.g. `sum(properties.price)`) as columns.
+
+This is helpful not just for generating tables which summarize person and event information according to your needs, but also for exploring the data in new and totally custom ways. You could, for example, use HogQL to run currency conversions and normalize data into a single currency in its own column - or even extrapolate and project revenue and usage figures into the future.
+
+## Custom insights
+![custom insights](../images/blog/array/custom_insights.gif)
+
+Finally, we're introducing custom insights, which enable you to create a very wide range of insights and visualizations. 
+
+Custom insights basically turn any list of events or persons - almost every table in PostHog, basically - into an insight which can be added to a dashboard. 
+
+You can add tables directly by hitting the new 'Open as a new insight' button (above) to reveal the source behind the table, and editing it to create new views. You can also also use the 'Edit source' button on existing insights to accomplish the same. 
```

Comment:
```suggestion
You can add tables directly by hitting the new 'Open as a new insight' button (above) to reveal the source behind the table, and editing it to create new views. You can also also use the 'View source' button on existing insights to accomplish the same. 
```

I think "edit source" makes some more sense, but the button is currently called "View source", so 🤷